### DISCUSSION TITLE
"No Selected Project"

### DISCUSSION THREAD

**Discussion by Jase-Stamets** [rfvgyhn/min-ed-launcher#133] [Q&A] [answered]
Hey, sorry I'm relatively new to messing with Elite Dangerous. Normally I played on console but branching out to PC now that I have one worth a damn. I love the idea of this, especially being able to autoboot a few other things with it, but the basic installation is presenting me with this error. Any ideas? Sorry if I'm being a complete idiot. I've looked over the set up and read me a bit and I couldn't find any obvious fault but knowing me it is something PAINFULLY obvious. If so, I apologize

![image](https://github.com/rfvgyhn/min-ed-launcher/assets/172307612/aad9475a-da59-4f19-b2d6-a14f8eb84217)

**Comment by demonbane** [comment]
Have you tried launching it at least once without min-ed-launcher just to make sure it works with the built in launcher?

**Reply by Jase-Stamets** [reply]
Oh yeah I've been playing for hours. The game works fine through it's own default launcher and no added Steam settings. Sorry, did forget to mention it was the Steam version but thats probably obvious from the screenshot at this point.

**Comment by rfvgyhn** [comment, marked as answer]
It seems like you've specified `/autorun` without specifying which product to run or you specified `/edo` but don't own it. You'll need to specify `/edh4` in order to launch Horizons (Live)

If that's not it, can you provide output from your log file? `%LOCALAPPDATA%\min-ed-launcher\min-ed-launcher.log`

**Reply by Jase-Stamets** [reply]
I'm just going to quietly go and throw myself off of a cliff...

**Reply by rfvgyhn** [reply]
Hahah. It's an easy mistake to make and only a two character difference so it can be hard to spot. The error message isn't very helpful either.

**Comment by fogbunny** [comment]
![MinEdLauncher_HWjxT4I5h1](https://github.com/user-attachments/assets/a75e59e7-1acc-471a-9a21-83cf122aa7f9)
this is mine what should i write?

**Comment by sbr992** [comment]
Anyone here later with problems with Epic and not installed the game yet and getting this error. Took me a lot of googling to find this answer: https://github.com/rfvgyhn/min-ed-launcher/discussions/142

It helped me when I had no clue, hopefully it'll help you as it doesn't come up under normal search results.
